### PR TITLE
minor adjustments

### PR DESCRIPTION
**Pull request by bradymiller**
@sjpadgett , also gotta remember to bring this into rel-700 after your ccda fixes go there

### CODE REVIEW

**Comment by bradymiller** [comment]
yep, you are correct, the limit was escaped in several lines above. will revert that fix. my eagle eyes are dimming :)

File: interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncountermanagerTable.php [hunk 1]
```diff
@@ -339,7 +339,7 @@ public function getFileID($pid, $limit = 1)
         $query = "SELECT cc.id, pd.fname, pd.lname, pd.pid FROM ccda AS cc
 		    LEFT JOIN patient_data AS pd ON pd.pid=cc.pid
 		    WHERE cc.pid = ?
-		    ORDER BY cc.id DESC LIMIT $limit";
+		    ORDER BY cc.id DESC LIMIT " . escape_limit($limit);
```
Comment (sjpadgett) [inline interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncountermanagerTable.php]: escaped above

Comment (bradymiller) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncountermanagerTable.php]: yep, double escaped :)
maybe should do following to really be sure :)
```
ORDER BY cc.id DESC LIMIT ";
for ($i=0; $i<1000000000; $i++) {
    $limit = escape_limit($limit);
}
$query .= $limit;
```

Comment (sjpadgett) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncountermanagerTable.php]: I tend to try to set variables before using.

File: src/Services/Cda/CdaTemplateImportDispose.php [hunk 2]
```diff
@@ -2033,14 +2033,13 @@ public function findClosestEncounter($item_date, $item_pid)
      */
     public function findClosestEncounterWithForm($item_date, $item_pid, $form = '')
     {
-        $form = add_escape_custom($form);
```
Comment (bradymiller) [inline src/Services/Cda/CdaTemplateImportDispose.php:2036]: @sjpadgett , let me know if you want me to change above to (would keep with your style of escaping variables prior):
```
$form = escape_table_name($form);
```
(and then I would remove the line below)

Comment (sjpadgett) [reply src/Services/Cda/CdaTemplateImportDispose.php:2036]: No you're fine.

**Pull request merged**